qiskit/transpiler/passes/synthesis/hls_plugins.py

```diff
@@ -262,15 +262,46 @@
    MCMTSynthesisNoAux
    MCMTSynthesisDefault
 
+
+Pauli Evolution Synthesis
+'''''''''''''''''''''''''
+
+.. list-table:: Plugins for :class:`.PauliEvolutionGate` (key = ``"PauliEvolution"``)
+    :header-rows: 1
+
+    * - Plugin name
+      - Plugin class
+      - Targeted connectivity
+    * - ``"rustiq"``
+      - :class:`~.PauliEvolutionSynthesisRustiq`
+      - all-to-all
+    * - ``"default"``
+      - :class:`~.PauliEvolutionSynthesisDefault`
```

Comment:
Just a note: you could skip these as they just truncate the path (but there is no path)
```suggestion
      - :class:`.PauliEvolutionSynthesisRustiq`
      - all-to-all
    * - ``"default"``
      - :class:`.PauliEvolutionSynthesisDefault`
```

Reply:
I actually know this one, but for some reason all of our plugin tables have the "~", including the one you have recently added for MCMT. :)

Reply:
Oh then I was too quick to copy+paste 😂 Fair enough, let's clean this up some time later